Makefile

```diff
@@ -11,7 +9,9 @@ rpi linux bebop edison:
 	cd build_$@ && make
 
 qurt:
-	$(call df_cmake_generate,$@,$@,cmake/cmake_hexagon/toolchain/Toolchain-$@.cmake,-DQC_SOC_TARGET=${QC_SOC_TARGET})
+	# qurt needs to be separate from rpi, bebop, etc. because it has "qurt"
```

Comment:
David Sidrane had issues setting the DF variable when building PX4 for NuttX.  I'm OK with the name changes for consistency.